src/tests.rs

```diff
@@ -242,7 +242,6 @@ mod cli {
         let db = open_db();
 
         db.create_table_if_missing();
-        db.clear_all_tasks();
```

Comment:
This line is specifc to the `clear_tasks`, it cant be deleted because there is a need to test if clearing all tasks works

Reply:
Ooops. I think you intended to insert the above-created task before clearing then?

Reply:
Yes that's right